aries-site/src/pages/components/index.js

```diff
@@ -1,33 +1,59 @@
 import React from 'react';
-import { Layout, NavPage } from '../../layouts';
-import { DescriptiveHeader, Meta, ComingSoon } from '../../components';
-import { getPageDetails } from '../../utils';
+import Link from 'next/link';
+import { Box, Heading, Paragraph } from 'grommet';
+
+import { CardGrid, ContentCard, Meta } from '../../components';
+import { structure } from '../../data';
+import { Layout, PageIntro } from '../../layouts';
+import { getPageDetails, getParentPage, nameToPath } from '../../utils';
 
 const title = 'Components';
-const page = getPageDetails(title);
+const pageDetails = getPageDetails(title);
 
-const Components = () => {
-  const descriptiveHeader = (
-    <DescriptiveHeader
-      background={page.color}
-      subText={page.description}
-      icon={page.icon}
-      title={title}
-    />
-  );
+const cards = structure
+  .map(obj => {
+    const page = obj;
+    const parent = getParentPage(page.name);
+    page.parent = parent;
+    return page;
+  })
+  .filter(page => page.parent !== undefined)
+  .filter(page => page.parent.name === title);
 
+const Components = () => {
   return (
-    <Layout descriptiveHeader={descriptiveHeader} title={title} isNavPage>
+    <Layout title={title} isLanding>
       <Meta
         title={title}
-        description={page.seoDescription}
-        canonicalUrl="https://design-system.hpe.design/components"
+        description={pageDetails.seoDescription}
+        canonicalUrl="https://design-system.hpe.design/templates"
       />
-      {page.pages.length ? (
-        <NavPage items={page.pages} topic={page.name.toLowerCase()} />
-      ) : (
-        <ComingSoon />
-      )}
+      <Box gap="large">
+        <PageIntro
+          image={{
+            src: '/card-components.png',
```

Comment:
Can we do svg instead? Feels like the png is getting a little pixelated, so svg might be better since it's vectors

Reply:
We can, just had to use what was handy from Chris at the moment. Time crunch.

Reply:
I'll see if I can break away tonight to get a few more edits in.

Reply:
I exported from the file as SVG -- I can help if needed.

Reply:
Thanks for the offer. I'll take care of now that we have a little extra time. :)

Reply:
@taysea Implemented.